### ISSUE TITLE
Taskbar Colour Permanantly Stuck

### ISSUE THREAD

**Issue by jahinzee** [Open-Shell/Open-Shell-Menu#567]
When using Open-Shell to change the taskbar colour, the taskbar suddenly stopped responding to any further changes to the taskbar settings, and is permanantly stuck on Taskbar colour #FFFFFF.

The taskbar text colour can still be changed, but the actual colour and transparency of the taskbar is stuck, and cannot be changed.

This issue persists even if Open-Shell menu is quit, Windows is restarted, or Open-Shell is uninstalled.

**To Reproduce**
1. Right-Click Start button
2. Click "Settings"
3. Check "Show all settings"
4. Click "Taskbar"
5. Changing the "Taskbar color" and "Taskbar opacity" settings; it worked fine

**Screenshots**
![image](https://user-images.githubusercontent.com/56011682/101886557-c7308380-3bef-11eb-83a8-a9e8ad917f7b.png)
The taskbar should be a plain black colour, but it is stuck on white with no way to change it.

**Version:**
 Open-Shell version `4.4.160`
 Running on Windows 10 Pro, Build `19041.vb_release.191206-1406`

**Comment by jahinzee** [comment]
I have figured out the issue myself _literally 2 minutes after posting_. It's been bugging me for days now. ¯\\\_(ツ)_/¯¯

The issue stems from another app I had installed, TransluscentTB. Apparently that app had closed incorrectly, and has been conflicting with Open-Shell's taskbar colouring. After reconfiguring TransluscentTB, the issue went away.

**Moral of the story?** Don't have two different theming utilities for the same element installed on the same system.
